xonsh/built_ins.py

```diff
@@ -351,8 +351,39 @@ def run_subproc(cmds, captured=True):
             stdin = PIPE
         else:
             stdin = prev_proc.stdout
-        proc = Popen(aliased_cmd, universal_newlines=uninew, env=ENV.detype(),
-                     stdin=stdin, stdout=stdout)
+        try:
+            proc = Popen(aliased_cmd, universal_newlines=uninew, env=ENV.detype(),
+                         stdin=stdin, stdout=stdout)
+        except FileNotFoundError:
+            cmd = aliased_cmd[0]
+            print('xonsh: subprocess mode: command not found: {0}'.format(cmd))
```

Comment:
Ummhmm.  I think tools.py is fine.  If it gets much beyond a handful of functions, then a new module is worth it. For now, though, I don't think it is needed.